bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -112,10 +119,16 @@ public LedgerFragmentReplicator(BookKeeper bkc, StatsLogger statsLogger) {
         numBytesWritten = this.statsLogger.getOpStatsLogger(NUM_BYTES_WRITTEN);
         readDataLatency = this.statsLogger.getOpStatsLogger(READ_DATA_LATENCY);
         writeDataLatency = this.statsLogger.getOpStatsLogger(WRITE_DATA_LATENCY);
+        if (conf.getReplicationRateByBytes() > 0) {
+            this.replicationThrottle = new Throttler(conf.getReplicationRateByBytes());
+        }
+        totalReplicatedEntriesSize = 0;
+        replicatedEntriesNum = 0;
+        averageEntrySize = 1024;
```

Comment:
`1024` should be define as constant

Reply:
`throttle.acquire` need permits larger than 0, 1024 is an initial value. no special reason.